conf/app.transportEquipment.index.routes

```diff
@@ -1,15 +1,15 @@
+POST       /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/items                                             controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index,goodsReferenceIndex: Index, mode: Mode = NormalMode)
+GET        /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/change-items                                      controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index,goodsReferenceIndex: Index, mode: Mode = CheckMode)
+POST       /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/change-items                                      controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index,goodsReferenceIndex: Index, mode: Mode = CheckMode)
 
 # These routes must come before the ApplyAnotherItemController routes due to regex matching
-GET        /:arrivalId/transport-equipment/:equipmentIndex/items/apply-another                                                    controllers.transportEquipment.index.ApplyAnotherItemController.onPageLoad(arrivalId: String, mode: Mode = NormalMode, equipmentIndex: Index)
-POST       /:arrivalId/transport-equipment/:equipmentIndex/items/apply-another                                                    controllers.transportEquipment.index.ApplyAnotherItemController.onSubmit(arrivalId: String, mode: Mode = NormalMode, equipmentIndex: Index)
-GET        /:arrivalId/transport-equipment/:equipmentIndex/items/change-apply-another                                             controllers.transportEquipment.index.ApplyAnotherItemController.onPageLoad(arrivalId: String, mode: Mode = CheckMode, equipmentIndex: Index)
-POST       /:arrivalId/transport-equipment/:equipmentIndex/items/change-apply-another                                             controllers.transportEquipment.index.ApplyAnotherItemController.onSubmit(arrivalId: String, mode: Mode = CheckMode, equipmentIndex: Index)
+GET        /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/items/apply-another                               controllers.transportEquipment.index.ApplyAnotherItemController.onPageLoad(arrivalId: String, mode: Mode = NormalMode, equipmentIndex: Index,goodsReferenceIndex: Index)
+POST       /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/items/apply-another                               controllers.transportEquipment.index.ApplyAnotherItemController.onSubmit(arrivalId: String, mode: Mode = NormalMode, equipmentIndex: Index,goodsReferenceIndex: Index)
+GET        /:arrivalId/transport-equipment/:equipmentIndex/:goodsReferenceIndex/items/change-apply-another                        controllers.transportEquipment.index.ApplyAnotherItemController.onPageLoad(arrivalId: String, mode: Mode = CheckMode, equipmentIndex: Index,goodsReferenceIndex: Index)
```

Comment:
After taking a proper look, I don't think we should be passing through the goodsReferenceIndex into the applyAnotherItemController. We want to show all goodsReferences applied to that index. Looking at applyAnotherItemViewModel in IE170, we can see that we can get the index required for the GoodsReferenceController without having it here